app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesComposables.kt

```diff
@@ -82,7 +85,13 @@ fun PocketStory(
             TabTitle(text = story.title, maxLines = 3)
         },
         subtitle = {
-            TabSubtitleWithInterdot(story.publisher, "${story.timeToRead} min")
+            if (isValidPublisher && isValidTimeToRead) {
```

Comment:
I would consider moving `isValidPublisher` and `isValidTimeToRead` into this content block.

Reply:
The values are not going to change for this `PocketStory` and above is also the `imageUrl` in sort of a prepare data -> setup composable arrangement.